### PR TITLE
Add password and encryption to private key

### PR DESCRIPTION
**Pull request by lego963**

### CODE REVIEW

File: scenarios/modules/bastion/variables.tf [hunk 1]
```diff
@@ -20,6 +20,6 @@ variable "volume_bastion" {
   type    = number
   default = 10
 }
-variable "scenario" {}
+variable "scenario" { default = "" }
```
Comment (outcatcher) [inline scenarios/modules/bastion/variables.tf]: Why was this added?

File: scenarios/core/get_key.py [hunk 2]
```diff
@@ -28,10 +29,12 @@ def generate_private_key():
         public_exponent=65537,
         key_size=2048
     )
+    password = input("enter password: ")
```
Comment (outcatcher) [inline scenarios/core/get_key.py]: Mix of `"` and `'`, please use `'` for this file

File: scenarios/core/get_key.py [hunk 3]
```diff
@@ -49,24 +50,31 @@ def get_key_from_s3() -> str:
...
             obj.put(Body=key)
             with open(output_file, 'wb') as file:
-                file.write(key)
+                file.write(get_decrypted_key(key, password))
             return output_file
         raise cl_e
 
 
+def get_decrypted_key(encrypted_key, password) -> str:
+    return crypto_serialization.load_pem_private_key(encrypted_key, password=password, backend=crypto_default_backend())
```
Comment (outcatcher) [inline scenarios/core/get_key.py]: line too long, please don't use lines >100 chars

File: scenarios/core/get_key.py [hunk 4]
```diff
@@ -49,24 +50,31 @@ def get_key_from_s3() -> str:
...
             obj = obs.Object(BUCKET, key_name)
             obj.put(Body=key)
             with open(output_file, 'wb') as file:
-                file.write(key)
+                file.write(get_decrypted_key(key, password))
             return output_file
         raise cl_e
 
 
+def get_decrypted_key(encrypted_key, password) -> str:
```
Comment (outcatcher) [inline scenarios/core/get_key.py]: One-line single-use function, maybe it's redundant?

**Comment by lego963** [comment]
Close #159
